js/src/util.js

```diff
@@ -42,7 +42,7 @@ const Util = (($) => {
         if ($(event.target).is(this)) {
           return event.handleObj.handler.apply(this, arguments) // eslint-disable-line prefer-rest-params
```

Comment:
IIRC, I was just too scared to touch that at the time. No objection to changing as long as it doesn't break anything.